src/mem/cache/Cache.py

```diff
@@ -111,6 +111,12 @@ class BaseCache(ClockedObject):
     replacement_policy = Param.BaseReplacementPolicy(
         LRURP(), "Replacement policy"
     )
+    partitioning_policies = VectorParam.BasePartitioningPolicy(
+        [],
+        "Partitioning policies"
+        "Setting multiple policies will enforce all of them individually \
+        in order",
```

Comment:
```suggestion
        "Setting multiple policies will enforce all of them individually "
        "in order",
```

The backslash is not needed. You can have a python string go across multiple lines. All of the lines will be concatenated together.

Reply:
Thank you for pointing this out - I've addressed this in [commit c30d384](https://github.com/gem5/gem5/pull/765/commits/c30d38426a7063695ad1418bc76e25d3e3e310c1)